app/models/alpha_sms_delivery_detail.rb

```diff
@@ -17,6 +17,10 @@ def in_progress?
     request_status.blank?
   end
 
+  def result
+    request_status
+  end
```

Comment:
```suggestion
alias result request_status
```

Reply:
Adding an alias won't work because `request_status` is not a static attribute or a method, it's added at runtime. The code fails with error: `NameError: undefined method `request_status' for class `#<Class:0x0000000148e7eef8>'`